PHP/Complex Flow Examples/decrypt-add-reencrypt.php

```diff
@@ -15,6 +15,13 @@
 * lock it up again.
 */
 
+// By default, we use the US-based API service. This is the primary endpoint for global use.
+$apiUrl = "https://api.pdfrest.com";
+
+/* For GDPR compliance and enhanced performance for European users, you can switch to the EU-based service by uncommenting the URL below.
+ * For more information visit https://pdfrest.com/pricing#how-do-eu-gdpr-api-calls-work
+ */
+//$apiUrl "https://eu-api.pdfrest.com";
```

Comment:
Applies to all PHP samples

Missing assignment `=`
```suggestion
//$apiUrl = "https://eu-api.pdfrest.com";
```